lib/src/migrators/calc_interpolation.dart

```diff
@@ -37,10 +37,16 @@ class _CalculationInterpolationVisitor extends MigrationVisitor {
     const calcFunctions = ['calc', 'clamp', 'min', 'max'];
     final interpolation = RegExp(r'\#{\s*[^}]+\s*}');
     final hasOperation = RegExp(r'[-+*/]+');
+    final isVarFunc = RegExp(r'var\(\#{\s*[^}]+\s*}\)');
```

Comment:
1. I don't think the `#` needs to be escaped here.
2. This regex won't match something like `var(--prefix-#{$variable}-suffix)`

Reply:
👍